Formula/hound.rb

```diff
@@ -0,0 +1,17 @@
+require "formula"
+
+class Hound < Formula
+  homepage "https://github.com/thoughtbot/hound-cli"
+  url "https://github.com/thoughtbot/hound-cli/archive/v0.1.0.tar.gz"
+  version "0.1.0"
+  sha256 "88486a5151be3c0be0515111781b7be5a2f4621e38b1d211d0c3671ad78f7e34"
+
+  def install
+    prefix.install "lib" => "lib"
```

Comment:
I think that this will symlink, but that if you change this name to `rubyLib` (like we do in `Liftoff`) that won't be the case. I think the `lib` directory is special cased for Homebrew.